gcc/rust/checks/errors/borrowck/rust-bir-builder-internal.h

```diff
@@ -75,7 +75,6 @@ struct BuilderContext
 
   // External context.
   Resolver::TypeCheckContext &tyctx;
-  Resolver::Resolver &resolver;
```

Comment:
We should probably store the NR2.0 resolver instead to avoid having to get it each time we want to use it